xray/commands/offlineupdate/offlineupdate.go

```diff
@@ -379,14 +391,27 @@ func getFilesList(updatesUrl string, flags *OfflineUpdatesFlags) (vulnerabilitie
 	return
 }
 
+// ValidStreams represents the valid values that can be provided to the 'stream' flag during offline updates.
+var ValidStreams = map[string]bool{PublicData: true, ContextualAnalysis: true, Exposures: true}
+
+const (
+	PublicData         string = "public_data"
+	ContextualAnalysis string = "contextual_analysis"
+	Exposures          string = "exposures"
+)
+
+func GetValidStreamsList() string {
+	return PublicData + ", " + Exposures + " and " + ContextualAnalysis
+}
```

Comment:
I suggest having the above map, consts and function encapsulated into one struct named `ValidStreams`.
The consts and the function can then be access through this struct. Also, the string returned by the `GetValidStreamsList ` can be created dynamically by the map, instead of having the const values duplicated.